src/index.html

```diff
@@ -9,7 +9,7 @@
     <meta content="<%= htmlWebpackPlugin.options.meta[key] %>" name="<%= key %>">
     <% } %>
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-    <link href="/bootstrap.min.css" rel="stylesheet" type="text/css">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
   </head>
```

Comment:
> Does the bootstrap link you provided limit it to version 3.3.7 whereas the other one is updated to the most recent version?

Yes, we will have to manually update the CDN link for version updates, whereas the other method would at least give us automatic patch-level updates. However, this was the easiest way to get the `webpack-dev-server` working. I can template which link we use based on environment if everyone thinks this is important.

> is this to speed up content loading?

That was not the intention, but yes, using a CDN will speed up the load time for that script